### ISSUE TITLE
search issue

### ISSUE THREAD

**Issue by JHSPHshe** [julesjanssen/chosen#54]
It works just fine few days ago, but now when i search word from middle sentence, i can only search one word instead of few word with space separator.

for example:

Atlanta Falcons
New Orleans Saints
Tampa Bay Buccaneers

If i search from beginning, everything works fine, but if i search "Bay Bucc" , it will show me 'no result match "Bay Bucc".

Any idea? thanks.

We use 1.1.0 chosen.jquery.min.js, in every browser

**Comment by allandesa** [comment]
Hi there, 
Search for winnow_resulsts and change the regex expression, remove the '^'+

Change "new RegExp('^' + searchText.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, "\\$&"), 'i'), "  to new RegExp(searchText.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, "\\$&"), 'i'),

**Comment by JHSPHshe** [comment]
which file i should going to search for? i search string "'^'" in 1.1.0/chosen.jquery.min.js file, but can not find the expression you mention above, any idea? thanks.

**Comment by julesjanssen** [comment]
See https://github.com/julesjanssen/chosen/blob/master/Source/chosen.js#L618-L619
But it looks like you're looking at the original jQuery version, not this MooTools fork.

**Comment by JHSPHshe** [comment]
Yes, we use original JQuery version (1.1.0 chosen.jquery.min.js), if so,how to solve the issue? thanks.

**Comment by julesjanssen** [comment]
No idea, but please have a look at that version's repository & documentation here: https://github.com/harvesthq/chosen
